### ISSUE TITLE
Clear FileSystemPersister

### ISSUE THREAD

**Issue by Shakezulla57** [NYTimes/Store#365]
I'm currently using the `FileSystemPersister` setup like this:

`FileSystemPersisterFactory.create(
        cache,
        pathResolver,
        1,
        TimeUnit.HOURS
    )
`

I'd like to use `getRefreshing()` and `clear()`. However, `FileSystemPersister` doesn't implement `Clearable`. How can I bust the cache of the `FileSystemPersister` while still utilizing a timed cache and `networkBeforeStale()`? 

The readme section about SourcePerster and FileSystem make it seem like this is supposed to be working. Am I missing something?

**Comment by digitalbuddha** [comment]
You are correct, the `FileSystemPersisterFactory` is a convenience factory for common persisters. If you'd like a persister that implements `Clearable` you would have to implement that yourself.  You can take a look at the persister that is generated by the factory as a starting point and then add the `Clearable` interface to it
